examples/show_trust.py

```diff
@@ -0,0 +1,36 @@
+from fapolicy_analyzer import *
+import argparse
+import sys
+from datetime import datetime
+
+
+def main(*argv):
+    parser = argparse.ArgumentParser()
+    parser.add_argument("trust_type", nargs="+", choices=["system", "file"], help="trust type to show")
+    parser.add_argument("-c", "--count", action='store_true', help="only show the count")
+    parser.add_argument("-t", action='store_true', help="show last modified time")
```

Comment:
Suggestion: Add a long option for outputing time. Maybe "--time" or something to that effect?